src/dashboards/Compact/styles.scss

```diff
@@ -153,6 +118,21 @@
     }
 }
 
+@media (min-width: 996px) {
+    .compact {
+        &__tiles {
+            /*
+            react-grid-layout uses a margin of 32px between all grid-items.
```

Comment:
Fint at du forklarar dette. 👍

Reply:
.Kanskje du kan benytte rem i stedet for px:

<img width="882" alt="Screenshot 2021-07-14 at 12 14 52" src="https://user-images.githubusercontent.com/67413374/125605428-d9ad5784-7236-4360-8431-2a58602e1ef5.png">